uwblueprint/social-diversity-for-children
# Implement admin browse programs page

### Notion ticket link



[Admin Browse Programs Page](https://www.notion.so/uwblueprintexecs/6dd603d3687743ce8d09d6298dc99748?v=7332b6570bcc4b48b5cd73fa2719bec8&p=f8bfd5cc486a46999d9676d731b5156f)



### Implementation description

- ticket is split into browse programs/browse classes. This PR is for browse programs page
- created new page /admin/programs to view all available programs
- displays all the programs through a card in a 3 column grid
- implemented a filter feature using the search bar to only display the program card if the name/description matches the filter query
- if no programs match the filter, an empty state is returned 

https://user-images.githubusercontent.com/71860803/141852436-6997d56e-104b-48b7-9bc3-3314e227b3d2.mp4





### Steps to test

1. Test that all the programs appear
2. Type into the search bar to filter through program names and their descriptions



### What should reviewers focus on?

### Checklist
